lcmtypes/lcmt_atlas_command.lcm

```diff
@@ -0,0 +1,58 @@
+package drake;
```

Comment:
Is the plan that the atlasFrames will use these types and the drcFrames will use drc types (i.e., the new types you have made will be duplicated in drc)?